benchmarks/annotation_store.ipynb

```diff
@@ -0,0 +1,2103 @@
+{
```

Comment:
<p>Please can we add TIAToolbox notebooks preamble. For example, link to open in colab/kaggle and some standard description at the top?</p><p> </p><p>I would also suggest to write a summary of why you have created this benchmark notebook. Probably copy some text from the manuscript.</p><p> </p><p>Also is it possible to measure these tests on colab and report?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TissueImageAnalytics/tiatoolbox/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
It is possible to run on colab but it's not easy to then save the output into a commit.

Reply:
Preamble added now (but won't work yet as it points to master).

Reply:
Added more introduction text and a mini tutorial for using the storage classes.